src/Nethermind/Nethermind.JsonRpc.Test/Modules/TestRpcModuleProvider.cs

```diff
@@ -31,7 +32,7 @@ namespace Nethermind.JsonRpc.Test.Modules
 {
     internal class TestRpcModuleProvider<T> : IRpcModuleProvider where T : class, IModule
     {
-        private RpcModuleProvider _provider = new RpcModuleProvider(new JsonRpcConfig(), LimboLogs.Instance);
+        private RpcModuleProvider _provider = new RpcModuleProvider(new FileSystem(), new JsonRpcConfig(), LimboLogs.Instance);
```

Comment:
I would pass just the regex to the module, not filesystem

Reply:
not much difference, this way we hide a bit more instead of loading the regex outside, since this is a less important, one off feature I prefer to keep it inside instead of pushing config out

Reply:
If not at least IEnumerable<string>, why RPC module would need file system?